app/Console/cake.php

```diff
@@ -11,36 +11,36 @@
  * Redistributions of files must retain the above copyright notice.
  *
  * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
  * @link          http://cakephp.org CakePHP(tm) Project
- * @package       app.Console
  * @since         CakePHP(tm) v 2.0
+ *
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
-
 if (!defined('DS')) {
-	define('DS', DIRECTORY_SEPARATOR);
+    define('DS', DIRECTORY_SEPARATOR);
 }
 
-$dispatcher = 'Cake' . DS . 'Console' . DS . 'ShellDispatcher.php';
+$dispatcher = 'Cake'.DS.'Console'.DS.'ShellDispatcher.php';
 
 if (function_exists('ini_set')) {
-	$root = dirname(dirname(dirname(__FILE__)));
-	$appDir = basename(dirname(dirname(__FILE__)));
-	$install = $root . DS . 'lib';
-	$composerInstall = $root . DS . $appDir . DS . 'Vendor' . DS . 'cakephp' . DS . 'cakephp' . DS . 'lib';
+    $root = dirname(dirname(dirname(__FILE__)));
+    $appDir = basename(dirname(dirname(__FILE__)));
+    $install = $root.DS.'lib';
+    $composerInstall = $root.DS.$appDir.DS.'Vendor'.DS.'cakephp'.DS.'cakephp'.DS.'lib';
 
-	// the following lines differ from its sibling
-	// /lib/Cake/Console/Templates/skel/Console/cake.php
-	if (file_exists($composerInstall . DS . $dispatcher)) {
-		$install = $composerInstall;
-	}
+    // the following lines differ from its sibling
+    // /lib/Cake/Console/Templates/skel/Console/cake.php
```

Comment:
__[staging][code_sniffer]__
* Inline comments must end in full-stops, exclamation marks, or question marks